carla/__init__.py

```diff
@@ -4,6 +4,10 @@
 
 from ._logger import INFOFORMATTER
 
+import warnings
+
+warnings.filterwarnings("ignore", message=r"Passing", category=FutureWarning)
```

Comment:
FYI @Philoso-Fish , this one supresses the ugly:

```
/home/morty/dev/carla/env/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/morty/dev/carla/env/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
```
warnings

Reply:
This is really great 👍